src/node_crypto.cc

```diff
@@ -2314,6 +2318,28 @@ void SSLWrap<Base>::DestroySSL() {
 }
 
 
+template <class Base>
+void SSLWrap<Base>::SetSNIContext(SecureContext* sc) {
+  InitNPN(sc);
+  SSL_set_SSL_CTX(ssl_, sc->ctx_);
+
+  SetCACerts(sc);
+}
+
+
+template <class Base>
+int SSLWrap<Base>::SetCACerts(SecureContext* sc) {
+  int err = SSL_set1_verify_cert_store(ssl_, SSL_CTX_get_cert_store(sc->ctx_));
+  if (err != 1)
+    return err;
+
+  STACK_OF(X509_NAME)* list = SSL_dup_CA_list(
+      SSL_CTX_get_client_CA_list(sc->ctx_));
+  SSL_set_client_CA_list(ssl_, list);
```

Comment:
Can you add a comment explaining that `SSL_set_client_CA_list` takes ownership of the duplicate?  And maybe explain why you copy it from the `SSL_CTX` to the `SSL`?